tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -526,3 +528,48 @@ def test_fetch_monthly_template_usage_for_service_does_not_include_test_notifica
     )
 
     assert len(results) == 0
+
+
+@pytest.mark.parametrize("notification_type, count",
+                         [("sms", 3),
+                          ("email", 5),
+                          ("letter", 7)])
+def test_get_total_sent_notifications_for_day_and_type_returns_right_notification_type(
+        notification_type, count, sample_template, sample_email_template, sample_letter_template
+):
+    create_ft_notification_status(bst_date="2019-03-27", service=sample_template.service, template=sample_template,
+                                  count=count)
+    create_ft_notification_status(bst_date="2019-03-27", service=sample_email_template.service,
+                                  template=sample_email_template, count=count)
+    create_ft_notification_status(bst_date="2019-03-27", service=sample_letter_template.service,
+                                  template=sample_letter_template, count=count)
```

Comment:
i think these three lines should hard code the count to 3/5/7. as it currently reads, you're just setting everything to count=`x` and asserting that `x` things are returned each time

Reply:
Doh! I think that's what I meant to do, good catch.